spec/policies.yml

```
@@ -41,7 +41,7 @@ components:
 
 
           **Any policy data which already exists on the server but is not explicitly specified in the new policy file will be deleted!**."
-        operationId: "loadPolicy"
+        operationId: "replace"
```

```suggestion
        operationId: "replacePolicy"
```
without saying anything about policy and divorced from the context of this file, I'm not sure what this method does - wdyt about just appending the method names with `Policy`?

---

the CLI uses `load`, `replace`, and `update` for POST / PUT / PATCH - maybe we should be consistent here?

https://github.com/cyberark/conjur-api-python3/blob/7041119a7adbf385896f7ef3cfcd41541d7fc305/conjur/cli.py#L261-L266

---

When myself and @jonahx talked we were focused more on matching the terminology with what is listed in the official Conjur documentation. Which lists the three as `append`, `replace`, `update` respectively. Should we want to follow the documentation or the CLI when implementing?

---

@izgeri Re: appending `Policy`, we're purposefully avoiding that so as not to create stutter.  With Ellis's current naming, it will be used like (correct me if I'm wrong @telday ):  `Policies.replace(...)`

---

the most common way people load policy is via the CLI - I agree the names load/replace/update aren't the most intuitive, and I also suggested some changes during the CLI development. but that's what we went with, and we should be consistent here - this is important so that the clients are consistent with the CLI

sorry for the delay in responding, somehow I missed this notification.

---

basically I guess this comment can be resolved because this is already what I suggest, `replace`. I added a new comment below for `load` instead of `append`.